Modules/Scripted/LineProfile/LineProfile.py

```diff
@@ -0,0 +1,598 @@
+import vtk, qt, slicer
+from slicer.i18n import tr as _
+from slicer.i18n import translate
+from slicer.ScriptedLoadableModule import *
+
+#
+# LineProfile
+#
+
+class LineProfile(ScriptedLoadableModule):
+    """Uses ScriptedLoadableModule base class, available at:
+    https://github.com/Slicer/Slicer/blob/main/Base/Python/slicer/ScriptedLoadableModule.py
+    """
+
+    def __init__(self, parent):
+        ScriptedLoadableModule.__init__(self, parent)
+        self.parent.title = _("Line Profile")
+        self.parent.categories = [translate("Quantification")]
+        self.parent.dependencies = []
+        parent.contributors = ["Andras Lasso (PerkLab)"]
+        self.parent.helpText = _("""
+This module computes the intensity profile of a volume along a markups line or curve.
+Notes:
+1. Lines that are on the image boundary (outside the center of border voxels) may be assigned 0 value.
+2. Probed points within the image volume are linearly interpolated among adjacent voxel centers
+and then returned with the same data type as the probed image (e.g. rounded to integers if the image has integer data type).
+    """)
+        self.parent.helpText += self.getDefaultModuleDocumentationLink()
+        self.parent.acknowledgementText = _("""
+This file was originally developed by Andras Lasso (PerkLab) and was partially funded by CCO ACRU.
+""") # replace with organization, grant and thanks.
+
+#
+# LineProfileWidget
+#
+
+class LineProfileWidget(ScriptedLoadableModuleWidget):
+    """Uses ScriptedLoadableModuleWidget base class, available at:
+    https://github.com/Slicer/Slicer/blob/main/Base/Python/slicer/ScriptedLoadableModule.py
+    """
+
+    def setup(self):
+        ScriptedLoadableModuleWidget.setup(self)
+
+        self.logic = LineProfileLogic()
+
+        # Instantiate and connect widgets ...
+        uiWidget = slicer.util.loadUI(self.resourcePath("UI/LineProfile.ui"))
+        self.layout.addWidget(uiWidget)
+        self.ui = slicer.util.childWidgetVariables(uiWidget)
+
+        #
+        # input volume selector
+        #
+        self.inputVolumeSelector = self.ui.inputVolumeSelector
+        self.inputVolumeSelector.nodeTypes = ["vtkMRMLScalarVolumeNode"]
+        self.inputVolumeSelector.selectNodeUponCreation = True
+        self.inputVolumeSelector.addEnabled = False
+        self.inputVolumeSelector.removeEnabled = False
+        self.inputVolumeSelector.noneEnabled = False
+        self.inputVolumeSelector.showHidden = False
+        self.inputVolumeSelector.setMRMLScene(slicer.mrmlScene)
+        self.inputVolumeSelector.setToolTip("Pick the input to the algorithm which will be sampled along the line.")
```

Comment:
Reminder to mark all of these tooltip strings for translation:
```suggestion
        self.inputVolumeSelector.setToolTip(_("Pick the input to the algorithm which will be sampled along the line."))
```